1.9/usage/tutorials/create-service.md

```diff
@@ -16,7 +16,7 @@ This tutorial shows how to create and deploy a simple one-command service and a
     
    1. In the **SERVICE ID** field, enter a name for your service. 
    1. In the **Command** field, enter `sleep 10`.
-   1. In the **Container Image** field, enter a container image to use.
+   1. In the **Container Image** field, enter a Docker container image to use, or click **MORE SETTINGS > MESOS RUNTIME** to use the default Mesos containerizer.
```

Comment:
Let's either provide a docker image (e.g. alpine) to run the tutorial or ask the users to change the runtime.